From 03f1c3d38887803266ec4d5a820b08b01b2766d8 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 26 May 2021 15:34:37 +0900
Subject: [PATCH] cifsd: remove unneeded initialization of rc variable in
 ksmbd_crypt_message()
Git-commit: 03f1c3d38887803266ec4d5a820b08b01b2766d8
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Remove unneeded initialization of rc variable in ksmbd_crypt_message().

Reviewed-by: Dan Carpenter <dan.carpenter@oracle.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/auth.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/cifsd/auth.c b/fs/cifsd/auth.c
index 9af1b334be82..711f8dec38e1 100644
--- a/fs/cifsd/auth.c
+++ b/fs/cifsd/auth.c
@@ -1236,7 +1236,7 @@ int ksmbd_crypt_message(struct ksmbd_conn *conn, struct kvec *iov,
 	struct smb2_transform_hdr *tr_hdr =
 		(struct smb2_transform_hdr *)iov[0].iov_base;
 	unsigned int assoc_data_len = sizeof(struct smb2_transform_hdr) - 24;
-	int rc = 0;
+	int rc;
 	struct scatterlist *sg;
 	u8 sign[SMB2_SIGNATURE_SIZE] = {};
 	u8 key[SMB3_ENC_DEC_KEY_SIZE];
-- 
2.33.0

